clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookMessageHandler.ts

```diff
@@ -134,3 +149,43 @@ export function downloadCopybookHandler(
     quietMode,
   );
 }
+
+async function getE4ECopyBookLocation(
+  copybookName: string,
+  documentUri: string,
+  storagePath: string,
+) {
+  const config = await E4ECopybookService.getE4EClient(documentUri, undefined);
```

Comment:
Now, this is a place where we definitely want to have the logging available, so provide a valid channel.

Reply:
This is the place we don't want to logging available. because logging won't trigger here.

Reply:
As far as I can tell, this function will end up calling the e4e api at least on the first resolve request.